peer.h

```diff
@@ -76,6 +76,18 @@ static inline dtls_security_parameters_t *dtls_security_params_epoch(dtls_peer_t
   }
 }
 
+static inline dtls_security_parameters_t *dtls_security_params_read_epoch(dtls_peer_t *peer, uint16_t epoch)
```

Comment:
I suggest a comment that explains this function's behavior as it requires at least careful reading.

Reply:
OK.